README.md

```diff
@@ -22,17 +22,42 @@ We are using [11ty](https://www.11ty.dev/docs/) and [Netlify](https://docs.netli
 - The site will be accessible on http://localhost:8080/en or http://localhost:8080/fr. You have to append the `/en` and `/fr` as there is no page available at the base url
   <br/>
 
-## Formatting
+## Adding new pages
+
+### Components
+
+To add a new component page, run the command `npm run create-component-page`. A prompt for English name and French name of the component will appear, continue by entering the name of the component in both languages. This will generate the directories `/src/en/components/<english-name>/` and `/src/fr/composants/<french-name>/` with `base.md`, `use-case.md`, `design.md` and `code.md` files in them.
```

Comment:
Should `English name` and `French name` be highlighted as code as well to make it stand out more as the prompt text?

Reply:
I think it will be okay if they are not highlighted.